src/core/transport/transport.h

```diff
@@ -72,6 +72,12 @@ typedef struct grpc_transport_stream_op {
 
   grpc_stream_op_buffer *recv_ops;
   grpc_stream_state *recv_state;
+  /** The number of bytes this peer is currently prepared to receive.
+
+      Bytes offered are used to replenish per-stream flow control windows.
```

Comment:
I prefer the version of this comment in src/core/transport/chttp2/internal.h. Would it make sense to just repeat that here ?